types/three/src/core/BufferGeometry.d.ts

```diff
@@ -11,7 +11,7 @@ import { EventDispatcher } from './EventDispatcher';
 import { BuiltinShaderAttributeName } from '../constants';
 
 /**
- * This is a superefficent class for geometries because it saves all data in buffers.
+ * This is a super efficient class for geometries because it saves all data in buffers.
```

Comment:
I think it's probably documentation in the era when they used to live together with `Geometry`.
If you are super generous and change the doc to the latest one I will appreciate it.

https://threejs.org/docs/#api/en/core/BufferGeometry

Reply:
Hello @0b5vr, I had already started working on improving TSDocs but I'm coding a way to do mostly automatic allowing update everything quicker. And this will take some time to complete.

So for now I just update the TSDocs for BufferGeometry in a very quick and dirty way. And in the near future, I will update with better TSDocs.

And a fix for all the returning types will be done in another PR.